tests/table.html

```diff
@@ -0,0 +1,9 @@
+<table>
+<tr><th>1st field </th><th>2nd field </th><th>3rd field </th></tr>
+<tr><td class="left">1st entry </td><td class="center">2nd entry </td><td class="right">3rd entry </td></tr>
+</table>
+<p>And here is another table</p>
+<table>
+<tr><th>Heading 1 </th><th>Some other heading </th></tr>
+<tr><td>I am a table cell. </td><td>Me too! </td></tr>
```

Comment:
Both in the heading and the entries, we retain any trailing white-spaces. Most other converters strip any trailing white-spaces.